test/lit/passes/type-generalizing.wast

```diff
@@ -274,4 +444,1128 @@
    )
   )
  )
+
+ ;; CHECK:      (func $call (type $2) (param $x eqref)
+ ;; CHECK-NEXT:  (local $var eqref)
+ ;; CHECK-NEXT:  (call $call
+ ;; CHECK-NEXT:   (local.get $var)
+ ;; CHECK-NEXT:  )
+ ;; CHECK-NEXT: )
+ (func $call (param $x eqref)
```

Comment:
There is already a test for a call on line 144, before this.

Reply:
I will rewrite that previous test to not use a call.

Reply:
Aha, no, the trick with those previous tests is that the call gets DCEd out, so actually they show nothing about how calls are analyzed. I could rewrite them to use `tuple.make` instead, but they're much simpler with the call, so I think it would be better to keep them as-is.

Reply:
Ok, sgtm. Maybe add a comment explaining that in those call usages?